Applications/MainApplication/Gui/MainWindow.cpp

```diff
@@ -1,8 +1,8 @@
 #include <Gui/MainWindow.hpp>
 #include <MainApplication.hpp>
 
-#include <Core/File/FileLoaderInterface.hpp>
-#include <Core/File/deprecated/OBJFileManager.hpp>
+#include <Core/Asset/FileLoaderInterface.hpp>
+#include <Core/Asset/OBJFileManager.hpp>

```

Comment:
Please keep deprecated classes as is ... so OBJFileManager has to be in a deprecated subdir (as far as I understand).

Reply:
Should namespaces be changed to match the new file structure too (Ra::Core::Asset::Deprecated) ?